ci/steps.lib.yml

```diff
@@ -130,7 +130,7 @@ config:
     dir: #@ input_name
     args:
     - "-c"
-    - "make $MAKE_HELP all run-tests"
+    - "make $MAKE_HELP all run-tests run-examples"
```

Comment:
If you accept my proposal to add a new target to `Makefile` to run all examples, then this would change to `test-examples`.

(That name makes sense to me given that this invocation is in CI scripts which are there for testing the system.)